robot/client/client.go

```diff
@@ -330,9 +330,10 @@ func (rc *RobotClient) connect(ctx context.Context) error {
 	if err := rc.connectWithLock(ctx); err != nil {
 		return err
 	}
-
+	rc.Logger().CInfow(ctx, "successfully reconnected to remote at address", "address", rc.address)
```

Comment:
```suggestion
	rc.Logger().CInfow(ctx, "successfully (re)connected to remote at address", "address", rc.address)
```
This will also get logged during initial connection establishment, no?